lib/engine/game/g_1856.rb

```diff
@@ -309,6 +311,33 @@ def cert_limit
         POST_NATIONALIZATION_CERT_LIMIT[num_corporations][@players.size]
       end
 
+      def destination_check
+        @corporations.each do |corp|
+          # The capitalization method of unparred corporations is nil
+          next unless corp.capitalization == :escrow
+
```

Comment:
start_hex, end_hex = @destinations[cop.id]

Reply:
I replaced this with `if hexes_connected?(*@destinations[corp.id])` and that worked